package-lock.json

```diff
@@ -1,12 +1,12 @@
 {
   "name": "cuttle",
-  "version": "5.1.3",
+  "version": "5.1.5",
   "lockfileVersion": 2,
   "requires": true,
   "packages": {
     "": {
       "name": "cuttle",
-      "version": "5.1.3",
+      "version": "5.1.5",
```

Comment:
Can't just change these here, running the version script also creates a tag so that may need to be addressed too. 

It's possible you needed to delete tag, reset the version and run it again. @itsalaidbacklife  we may need to test this case and add documentation on how to recover from it.

Reply:
Sorry just getting to this, I was curious about that. Ive never worked on another persons codebase so things like these I was curious of the possible problems that could happen. Yeah Ryan told me how to fix it. Appreciate you dudes